kitsune/lib/tlds.py

```diff
@@ -0,0 +1,6 @@
+import os
+
+path = os.path.join(os.path.dirname(__file__), 'tlds-alpha-by-domain.txt')
```

Comment:
Looks like it's updated every day: today https://data.iana.org/TLD/tlds-alpha-by-domain.txt has "Last Updated Wed Jun 24 07:07:02 2020 UTC" however no new TLDs were added.

Reply:
How do we intend to update that file (daily it's too much) but we should update it from time to time

Reply:
I considered including it as part of the build process, but then we have no way to manually validate it in case the format changes. I guess it'll cause the tests to fail if it does, so that might work.